pkg/config/config.go

```diff
@@ -22,9 +22,9 @@ type Config struct {
 
 	Mysql struct {
 		Host string `default:"notification-db"`
-		Port string `default:"3306"`
-		//Host     string `default:"192.168.0.10"`
-		//Port     string `default:"13306"`
+		Port int    `default:"3306"`
```

Comment:
int default should be 3306

Reply:
fieldSet(field *structs.Field, v string)
when parse the default value, the paramter should be string.